test/cluster/test_global_ignore_nodes.py

```diff
@@ -21,8 +21,8 @@ async def test_global_ignored_nodes_list(manager: ManagerClient, random_tables)
        since ignore node is permanent now and B is removed from the quorum early so it is enough to
        have two live nodes for the quorum.
     """
-    await manager.servers_add(2)
     servers = await manager.running_servers()
+    await manager.servers_add(2, property_file=[servers[1].property_file(), servers[2].property_file()])
```

Comment:
Moving `servers_add` below `servers = await manager.running_servers()` is incorrect because `servers` won't contain the two new servers anymore. CI probably failed because of this.